html/captive-portal/content/billing/authorizenet.js

```diff
@@ -0,0 +1,71 @@
+/* -*- Mode: javascript; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
+
+  var varsEl = $('#variables');
+  var vars = JSON.parse(varsEl.html());
+console.log(vars);
+
+
+  function sendPaymentDataToAnet() {
+      var secureData = {}; authData = {}; cardData = {};
+
+      // Extract the card number, expiration date, and card code.
+      cardData.cardNumber = document.getElementById("cardNumberID").value;
+      cardData.month = document.getElementById("monthID").value;
+      cardData.year = document.getElementById("yearID").value;
+      cardData.cardCode = document.getElementById("cardCodeID").value;
+      cardData.fullName = document.getElementById("name").value;
+      secureData.cardData = cardData;
+
+      // The Authorize.Net Client Key is used in place of the traditional Transaction Key. The Transaction Key
+      // is a shared secret and must never be exposed. The Client Key is a public key suitable for use where
+      // someone outside the merchant might see it.
+      authData.clientKey = vars.public_client_key;
+      authData.apiLoginID = vars.api_login_id;
+      secureData.authData = authData;
+
+console.log(secureData);
+
+      // Pass the card number and expiration date to Accept.js for submission to Authorize.Net.
+      Accept.dispatchData(secureData, responseHandler);
+
+      // Process the response from Authorize.Net to retrieve the two elements of the payment nonce.
+      // If the data looks correct, record the OpaqueData to the console and call the transaction processing function.
+      function responseHandler(response) {
+          if (response.messages.resultCode === "Error") {
+              for (var i = 0; i < response.messages.message.length; i++) {
+                  console.log(response.messages.message[i].code + ": " + response.messages.message[i].text);
+              }
+              alert("acceptJS library error!")
```

Comment:
It would be preferable to display an inline alert. Check how we do it in [content/billing/stripe.js](https://github.com/inverse-inc/packetfence/blob/devel/html/captive-portal/content/billing/stripe.js#L42-L46).